nebula/meta/ttypes.go

```diff
@@ -2118,7 +2118,7 @@ func (p *SpaceDesc) GetCollateName() []byte {
 }
 var SpaceDesc_VidType_DEFAULT *ColumnTypeDef = &ColumnTypeDef{
   Type:   7,
-  TypeLength: &const_lit_vid_type_type_length,
+  TypeLength: const_lit_vid_type_type_length,
```

Comment:
Maybe we should specify a default value manually here?

Reply:
I don't think it's necessary, this is the least modified.